share/meshes/write_exodus_file.m

```diff
@@ -1,75 +1,180 @@
-function write_exodus_file(coordx, coordy, coordz, tri, filename)
+function write_exodus_file_v2(filename, coordx, coordy, coordz, block, sidesets)
 
+% Number of vertices
 num_of_nodes = length(coordx);
+
+% Is the mesh 2D or 3D?
```

Comment:
Does this script actually support 3D meshes at this point? If not, maybe remove `num_of_dim` or just assign it to 2.

Reply:
The comment is misleading. The mesh elements are still 2D types but the vertices could be 2D or 3D. I will fix the comment.